android/src/main/kotlin/com/airship/flutter/AirshipPlugin.kt

```diff
@@ -31,6 +32,12 @@ private val TAG_OPERATION_ADD = "add"
 private val TAG_OPERATION_REMOVE = "remove"
 private val TAG_OPERATION_SET = "set"
 
+private val ATTRIBUTE_MUTATION_TYPE = "action"
```

Comment:
Would "ATTRIBUTE_MUTATION_ACTION" be clearer? If "ATTRIBUTE_MUTATION_TYPE" matches the other implementations, then don't worry about it.

Reply:
it's probably my fault, I made a comment on either the react or cordova implementation about the name because I found the additional overloading of "action" confusing. but the string it maps to ("action") remained the same.

Reply:
That's probably my fault, I made a previous comment on either the cordova or react implementation about how action was overloaded and confusing. but the string stayed the same. caveat emptor.

Reply:
"action" is defined by the api, the name of this constant isn't that big of a deal, I think changing this to type was reasonable